components/Workspaces/RepositoryStatCard.stories.tsx

```diff
@@ -20,6 +21,13 @@ export const PullRequests: Story = {
   },
 };
 
+export const Error: Story = {
```

Comment:
Adds an error state Storybook story for the repo stats cards.

![CleanShot 2024-01-30 at 17 06 23](https://github.com/open-sauced/app/assets/833231/399505e5-918d-4c53-a65f-11b98b67cd45)